lib/container_orchestrator.rb

```diff
@@ -0,0 +1,32 @@
+require 'kubeclient'
+
+class ContainerOrchestrator
+  TOKEN_FILE = "/run/secrets/kubernetes.io/serviceaccount/token".freeze
+
+  def scale(deployment_config_name, replicas)
+    connection.patch_deployment_config(deployment_config_name, { :spec => { :replicas => replicas } }, ENV["MY_POD_NAMESPACE"])
+  end
+
+  private
+
+  def connection
+    @connection ||=
+      Kubeclient::Client.new(
+        manager_uri,
+        :auth_options => client_auth_options,
+        :ssl_options  => { :verify_ssl => OpenSSL::SSL::VERIFY_NONE }
+      )
+  end
+
+  def manager_uri
+    URI::HTTPS.build(
+      :host        => ENV["KUBERNETES_SERVICE_HOST"],
+      :port        => ENV["KUBERNETES_SERVICE_PORT"],
+      :path        => "/oapi"
+    )
+  end
+
+  def client_auth_options
+    { :bearer_token_file => TOKEN_FILE }
```

Comment:
Feels like we don't need a method for this considering that the ssl_options don't have a method of their own.